lib/pointing_party_web/controllers/session_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule PointingPartyWeb.SessionController do
+  use PointingPartyWeb, :controller
+  alias PointingParty.Account.Auth
+  alias PointingParty.Account
+
+  def new(conn, _params) do
+    changeset = Account.changeset(%Account{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, params) do
+    case Auth.login(params["account"]) do
+      {:ok, %{username: username}} ->
+        put_session(conn, :username, username)
```

Comment:
One more small change here, can we start this pipeline with a raw value (`conn`)?

```elixir
conn
|> put_session(...)
|> redirect(...)
|> halt()
```